### DISCUSSION TITLE
Why is it called “Otterkit”?

### DISCUSSION THREAD

**Discussion by COBOL-Erik** [otterkit/otterkit-cobol#31] [General Discussion]
I haven’t seen or heard where the name comes from. Curious to find out!

**Comment by KTSnowy** [comment]
Hey @COBOL-Erik, the name was slightly inspired by SvelteKit (the *kit* part), we also wanted to make COBOL look a little more friendly and approachable so we chose an otter as a mascot. The end result was Otterkit, which together with the previous reasons could also be interpreted as "otter kit", meaning a baby otter.

Essentially we wanted to help improve the vision most people have of COBOL, of being an old dinosaur business language. Otters are cute and friendly so it helps with that.
